batch7crud-roll7/src/main/java/com/lftechnology/batch7crud/controller/StudentController.java

```diff
@@ -112,38 +116,37 @@ private void createProcess(HttpServletRequest request, HttpServletResponse respo
 
   private void edit(HttpServletRequest request, HttpServletResponse response, int id) throws ServletException, IOException, DataException {
     if(studentService.fetchById(id) == null) {
-      showNotFoundErrorPage(request, response);
-      return;
+      throw new ServletException(CommonConstants.PAGE_NOT_FOUND);
     }
     request.setAttribute("student", studentService.fetchById(id));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "student" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)